config/locales/views/submissions/nl.yml

```
@@ -24,6 +24,7 @@ nl:
       watch_submissions: Oplossingen automatisch herladen
       most_recent: Meest recente oplossing per gebruiker
       reevaluate_submissions: Oplossingen hertesten
+      detect_plagiarism: Detecteer plagiaat
```

```suggestion
      detect_plagiarism: Plagiaat detecteren
```

For consistency with the button next to it